src/__tests__/terminalCloudAPI.spec.ts

```diff
@@ -1,24 +1,34 @@
 import nock from "nock";
-import {createMockClientFromResponse, createTerminalAPIPaymentRequest} from "../__mocks__/base";
+import {createClient, createTerminalAPIPaymentRequest, createTerminalAPIRefundRequest} from "../__mocks__/base";
 import {asyncRes} from "../__mocks__/terminalApi/async";
-import {syncRes} from "../__mocks__/terminalApi/sync";
+import {syncRefund, syncRes} from "../__mocks__/terminalApi/sync";
 import Client from "../client";
 import TerminalCloudAPI from "../services/terminalCloudAPI";
 import {Convert, TerminalApiResponse} from "../typings/terminal";
 
-
 let client: Client;
 let terminalCloudAPI: TerminalCloudAPI;
 let scope: nock.Scope;
 
 beforeEach((): void => {
-    client = createMockClientFromResponse();
+    if (!nock.isActive()){
+        nock.activate();
+    }
+    client = createClient(process.env.ADYEN_TERMINAL_APIKEY);
+    client.config.merchantAccount = process.env.ADYEN_TERMINAL_MERCHANT;
+
     terminalCloudAPI = new TerminalCloudAPI(client);
     scope = nock(`${client.config.terminalApiCloudEndpoint}`);
 });
 
+afterEach((): void => {
+    nock.cleanAll();
+});
+
+const isCI = process.env.CI === "true" || (typeof process.env.CI === "boolean" && process.env.CI);
 describe("Terminal Cloud API", (): void => {
-    it("should make an async payment request", async (): Promise<void> => {
+    test.each([isCI, true])("should make an async payment request, isMock: %p", async (isMock): Promise<void> => {
```

Comment:
When isCI is true, doesn't that mean both cases being tested are true?
Shouldn't the second element of the array here perhaps be "!isCI" ?